src/assets/CollectibleDetectionController.test.ts

```diff
@@ -328,50 +346,79 @@ describe('CollectibleDetectionController', () => {
   });
 
   it('should not autodetect collectibles that exist in the ignoreList', async () => {
+    const selectedAddress = '0x2';
     collectibleDetection.configure({
       networkType: MAINNET,
       selectedAddress: '0x2',
     });
+    collectiblesController.configure({ selectedAddress });
+
+    const { chainId } = collectibleDetection.config;
+
     await collectibleDetection.detectCollectibles();
-    expect(collectiblesController.state.collectibles).toHaveLength(1);
+    expect(
+      collectiblesController.state.allCollectibles[selectedAddress][chainId],
+    ).toHaveLength(1);
     expect(collectiblesController.state.ignoredCollectibles).toHaveLength(0);
     collectiblesController.removeAndIgnoreCollectible(
-      '0x1d963688fe2209a98db35c67a041524822cf04ff',
+      '0x1d963688FE2209A98dB35C67A041524822Cf04ff',
```

Comment:
just curious why was the checksum change here necessary?

Reply:
Not really, I did it without noticing 😅